### DISCUSSION TITLE
Getting automatically rejected due to foreign documents

### DISCUSSION THREAD

**Discussion by JamesParkDev** [community/community#102047] [GitHub Education]
### Hi everyone,

Hello 👋 

I'm really having issues trying to upload my school contract as proof.

The website keeps telling me to edit my: Billing Information and to put: "full legal name". This I have done, and provided my full legal name that is visible on the document.

The issue is that the contract is in Serbian (Cyrillic version as all other legal documents are in Serbia)

I had this issues before, and ended up having to create tickets for manual review which were accepted for past few years, but GitHub removed this option and this makes it impossible for me to re-apply for my benefits.

I would really appreciate if I can get email of someone I can send me proof to or other ways of providing proof to GitHub.

Thanks.

**Comment by JamesParkDev** [comment]
Bump

**Reply by yashmalvade2002** [reply]
jbgrg

**Reply by JamesParkDev** [reply]
> jbgrg

What does this mean

**Comment by JamesParkDev** [comment]
I literally cannot verify I need support
